modules/bridge/src/main/ts/ephox/bridge/components/dialog/Tree.ts

```diff
@@ -11,12 +11,23 @@ export interface TreeSpec {
   type: 'tree';
   items: TreeItemSpec[];
   onLeafAction?: (id: Id) => void;
+  defaultExpandedIds?: Id[];
+  onExpand?: (
+    expandedKeys: Id[],
+    { expanded, node }: { expanded: boolean; node: Id }
+  ) => void;
 }
 
 export interface Tree {
   type: 'tree';
   items: TreeItem[];
+  defaultExpandedIds: Optional<Id[]>;
   onLeafAction: Optional<(id: Id) => void>;
+  onExpand: Optional<(
```

Comment:
Since this is both expanding and collapsing maybe it would be better to just have to functions here also let the tracking of these things be handled by the integrator instead it makes it harder to integrate but makes the api and code complexity simpler from our side. Having it be called when nodes are collapsing and calling it expand is confusing to me at least.
```
onExpand: (id: Id),
onCollapse: (id: Id)
```

Reply:
Didn't we change this policy to use only one function since tiny 6. For example in the forms we no longer have `api.disable(name)` and `api.enable(name)`, instead we have `api.setEnabled(boolean, name)` ? Also this functions provides the array of expanded ids so it is not just a boolean.

Reply:
That was because `disable` is politically incorrect. We could have a single function but then it should be called something like `onToggleExpand` or something since it's called when it collapsed as well.

Reply:
I see. I would rather call it `onToggleExpand`. I will change that. Thanks

Reply:
Done!